requirements/bootstrap.txt

```diff
@@ -0,0 +1,3 @@
+pip>=22.0.0
```

Comment:
What are your thoughts on adding an upper bound to these dependencies? Mainly I want to make sure that we are explicit in which dependencies the source build supports as it is not uncommon for them to break backwards compatibility.

Reply:
I think setuptools and pip are probably the two candidates for that as they break fairly regularly. Wheel is probably safe.